react-native-libraries.json

```diff
@@ -12202,5 +12202,30 @@
     "examples": ["https://github.com/nativeflowteam/nativeflowcss/tree/main/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/grahammendick/navigation/tree/master/build/npm/navigation",
+    "ios": true,
+    "android": true,
+    "web": true,
+    "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/grahammendick/navigation/tree/master/build/npm/navigation-react",
+    "examples": [
+      "https://github.com/grahammendick/navigation/tree/master/NavigationReact/sample"
+    ],
+    "ios": true,
+    "android": true,
+    "web": true
+  },
```

Comment:
should i add `newArchitecture` even for the dependencies like this one? `navigation-react-native` has all the stuff related to native stuff, not sure if to add new arch property to `navigation-react` as well, all it includes is just react code that works universally

Reply:
Thanks for the updates! 👍 

> should i add newArchitecture even for the dependencies like this one?

Yes, please include that tag, it would just confirm the fact, and would boost the library score a bit in the migration period.

Reply:
Done! Please take a look and please let me know if there is something else I might need to modify